### ISSUE TITLE
Nx Code Organization wiki page removed

### ISSUE THREAD

**Issue by ternyk** [nrwl/nx#585]
Hi
I can't see "Nx Code Organization" page in the wiki. It was present in the last week. Was it intentionally removed? If yes, why?

**Comment by electrichead** [comment]
I believe @ThomasBurleson had a page here before. He's away for a bit but it would be a good starting point.

**Comment by ThomasBurleson** [comment]
@ternyk - @vsavkin and I removed the old page for two important reasons:

1.  We needed to simplify the cognitive overhead needed to start organizing in Nx Workspaces. That draft was too complicated and needed refinements.
2.  We decided to publish these ideas in two (2) parts: a 2-3 page `Quick Start to Nx Workspaces` and deep-dive `Organizing your code in Nx Workspaces`. 

>  We are working of these docs and plan for public availability before August.
